joomla/joomla-cms
# [4.0][com_config] Fixed sending the test mail

### Summary of Changes
Currently when you want to send out a test mail you get a notice that it is not possible because mail sending is disabled, even though it is enabled. Added the lost language strings as well.

### Testing Instructions
1. Make sure you have a Joomla 4 installation
2. Click on System in the menu
3. Click on Global Configuration
4. Click on the Server tab
5. Make sure that Send Mail is enabled
6. Click on Send Test Mail
7. Notice you get a message saying that mail sending has been disabled:
<img width="647" alt="image" src="https://user-images.githubusercontent.com/359377/51425334-ea588100-1bda-11e9-962b-e63a1bdabc0d.png">

8. Apply the patch
9. Run the `npm ci` in the Joomla folder to generate the updated files
10. Click Send Test Mail
11. Notice that the test email has been sent
<img width="598" alt="image" src="https://user-images.githubusercontent.com/359377/51425416-3d7f0380-1bdc-11e9-9248-d50099cd7e96.png">


I may need help to correctly generate all JS files, so if there is any issues please let me know.

### Expected result
That the test email is send when mail sending is active


### Actual result
The test email is not send when mail sending is active


### Documentation Changes Required
None
